src/Intl/Grapheme/Grapheme.php

```diff
@@ -179,10 +179,17 @@ private static function grapheme_position($s, $needle, $offset, $mode)
         if ($offset > 0) {
             $s = self::grapheme_substr($s, $offset);
         } elseif ($offset < 0) {
-            if (\PHP_VERSION_ID < 50535 || (50600 <= \PHP_VERSION_ID && \PHP_VERSION_ID < 50621) || (70000 <= \PHP_VERSION_ID && \PHP_VERSION_ID < 70006)) {
```

Comment:
@nicolas-grekas why did you remove the version check here? I think this is the cause for the current failing tests.

Reply:
I don't remember :)